AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/utils.py

```diff
@@ -92,10 +97,8 @@ def fail_fast(message=None):
         raises FailFast
         :return: None
         """
-        Report.info("Failing fast. Raising an exception and shutting down the editor.")
         if message:
             Report.info("Fail fast message: {}".format(message))
-        TestHelper.close_editor()
```

Comment:
Is it safe to remove this? are other tests relying on this?

Reply:
No test should ever close the editor. This code is just wrong